main/src/ca/uwaterloo/flix/language/ast/TypedAst.scala

```diff
@@ -20,10 +20,15 @@ import java.lang.reflect.{Constructor, Field, Method}
 
 import ca.uwaterloo.flix.language.ast.Ast.{Denotation, Source}
 import ca.uwaterloo.flix.language.debug.{FormatExpression, FormatPattern}
+import ca.uwaterloo.flix.util.collection.MultiMap
+
+import scala.collection.immutable.List
 
 object TypedAst {
 
   case class Root(classes: Map[Symbol.ClassSym, TypedAst.Class],
+                  sigs: Map[Symbol.SigSym, TypedAst.Sig], // MATT check for ordering
```

Comment:
What do you mean ordering?

Reply:
A note to self to make sure the ordering of `classes`, `sigs`, `instances`, etc. in `*Ast.Root` is consistent.